### PR TITLE
Add Import/Export to Swagger Spec

### PR DESCRIPTION
**Pull request by nathannfan**
Add SQL Import/Export functionality to swagger spec with examples.

SDK PR: https://github.com/Azure/azure-sdk-for-net/pull/2887

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [ ] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

File: arm-sql/2014-04-01/swagger/importExport.json [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+                "x-ms-examples": {
+                    "Get Import/Export operations by database": { "$ref": "../examples/ImportExportOperationGetByDatabase.json" }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ImportExtensionProperties": {
+            "properties": {
+                "operationMode": {
```
Comment (anudeepsharma) [inline arm-sql/2014-04-01/swagger/importExport.json]: As this is always import, it is better to remove it in next API version. Please log a bug against the service.

File: arm-sql/2014-04-01/swagger/importExport.json [hunk 2]
```diff
@@ -0,0 +1,533 @@
...
+                    }
+                }
+            },
+            "required":["storageKeyType","storageKey","storageUri","administratorLogin","administratorLoginPassword"],
+            "description": "Export database parameters."
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+        "name": "subscriptionId",
```
Comment (anudeepsharma) [inline arm-sql/2014-04-01/swagger/importExport.json]: fix indentation for parameters

**Review by anudeepsharma — APPROVED** [review]
Looks good to me. Minor issue with indentation. Also to make apis cleaner for next version asking you to create backlog items in your service. Once indentation is fixed, let me know I can merge the PR.

File: arm-sql/2014-04-01/examples/DatabaseCreateImportMax.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "serverName": "sqlcrudtest-5961",
+    "databaseName": "testdb",
+    "api-version": "2014-04-01",
+    "parameters": {
+      "databaseName":"TestDbImport",
+      "edition":"Basic",
+      "serviceObjectiveName":"Basic",
+      "maxSizeBytes":"2147483648",
+      "storageKeyType":"StorageAccessKey",
+      "storageKey": "sdlfkjdsf+sdlfkjsdlkfsjdfLDKFJSDLKFDFKLjsdfksjdflsdkfD2342309432849328479324/3RSD==",
```
Comment (nathannfan) [inline arm-sql/2014-04-01/examples/DatabaseCreateImportMax.json]: Yes

File: arm-sql/2014-04-01/examples/DatabaseCreateImportMax.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "sqlcrudtest-4799",
+    "serverName": "sqlcrudtest-5961",
+    "databaseName": "testdb",
+    "api-version": "2014-04-01",
+    "parameters": {
+      "databaseName":"TestDbImport",
+      "edition":"Basic",
+      "serviceObjectiveName":"Basic",
+      "maxSizeBytes":"2147483648",
```
Comment (nathannfan) [inline arm-sql/2014-04-01/examples/DatabaseCreateImportMax.json]: I'm documenting all the bugs we come across and will bring them up when we discuss API changes

File: arm-sql/2014-04-01/swagger/importExport.json (hunk 2 above)
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/importExport.json]: Done

**Pull request merged**